packages/@aws-cdk/region-info/build-tools/pre-build.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+set -euo pipefail
+
+echo "⏳ Compiling static data generator..."
```

Comment:
why a custom build here?

Reply:
Because you cannot compile code that depends on generated code before the generated code was generated?